### DISCUSSION TITLE
WebGL Fingerprint

### DISCUSSION THREAD

**Discussion by ilyafedo10ko** [fingerprintjs/fingerprintjs#910] [Ideas]
Do you plan to create WebGL Fingerprint in addition to Canvas Fingerprint?

**Comment by Finesse** [comment]
No. We removed WebGL canvas image from entropy sources because it's too unstable.

**Reply by velheor** [reply]
Canvas image is still in use here https://fingerprintjs.github.io/fingerprintjs/, is it stable?
